docs/medical-api/api-reference/patient/bulk-create-patient.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: "Bulk Patient Create"
+description: "Creates multiple Patients in Metriport at once."
+api: "POST /medical/v1/patient/bulk"
+---
+
+The Bulk Patient Create flow allows you to insert up to 100,000 patients at Metriport in a
+single request/job. We'll also populate the medical data for those patients automatically.
+
+This feature avoids the complexity to work within the boundaries of
+[rate limits](/medical-api/more-info/limits#rate-limits) and to coordinate the data
+query for large groups of patients.
+
+<Info>
+  This is an asynchronous endpoint: you upload a file with
+  patient demographics and we will notify you when the
+  process is completed, through a [webhook
+  message](#webhook-with-the-result).
+</Info>
```

Comment:
we will notify you
=> we notify you

use present / infinitive tense in docs

Reply:
done